### DISCUSSION TITLE
Incoming / Outbound call icons for call history

### DISCUSSION THREAD

**Discussion by thomaschaaf** [twilio-labs/paste#2406] [Request: Icon] [answered]
I am building a call history for flex. It would be nice to have an icon showing an inbound / outbound call.

**Comment by TheSisb** [comment]
Related: https://github.com/twilio-labs/paste/discussions/2407

**Comment by SiTaggart** [comment, marked as answer]
Hi @thomaschaaf 

The CallIncoming and CallOutgoing icons are now available to use in Figma an NPM https://paste.twilio.design/components/icons
